digital_benchmark/twitter_benchmark/views.py

```diff
@@ -32,9 +37,74 @@ class SuccessView(View):
     def get(self, request):
         obj = DataProvider(request.GET['oauth_verifier'], request.GET['oauth_token'])
         # for testing
+        #ehmadzubair
         data = obj.get_other_tweet('ehmadzubair')
-        #data = TwitterDataParser.parser_other_tweet(data=data)
-        print(data)
+        data = TwitterDataParser.parser_other_tweet(data)
+        #print(data)
         return render(request, 'success/index.html')
 
 
+class UserDataList(generics.RetrieveAPIView):
+    permission_classes = [IsAuthenticated]
+
+    def retrieve(self, request, pk=None):
```

Comment:
override "get_queryset()" instead of "retrieve()". Read the difference between two methods.